front/src/Stores/ChatStore.ts

```diff
@@ -25,6 +30,11 @@ export interface ChatMessage {
     text?: string[];
 }
 
+export interface WritingMessage {
+    type: ChatMessageTypes;
```

Comment:
The type can actually only be "userWriting" here. Am I right?
It can never be `userStopWriting` (because we would remove the message from the list)

```suggestion
    type: ChatMessageTypes;
```